app/Services/ReportbackService.php

```diff
@@ -41,7 +41,7 @@ public function create($data, $transactionId)
 
         // POST reportback back to Phoenix, unless told not to.
         // If request fails, record in failed_jobs table.
-        if (! $data['do_not_forward']) {
+        if (! isset($data['do_not_forward'])) {
```

Comment:
yeah, but I think that is ok? because the only time it is being set is in the [migration script](https://github.com/DoSomething/phoenix/pull/7226/files#diff-9d15d4c1e9153ce45a524cc0e8900b4bR51) 
the rest of the values don't have that included...